contents/tutorials/free-hotjar-alternative.md

```diff
@@ -0,0 +1,74 @@
+---
+title: How to replace Hotjar for free with PostHog
+sidebar: Docs
+showTitle: true
+featuredImage: ../images/tutorials/banners/free-hotjar-alternative-banner.png
+featuredTutorial: true
+author: ["joe-martin"]
+topics: ['session recording']
+date: 2022-02-16
+---
+
+_Estimated reading time: 9 minutes_ ☕☕
+
+Hotjar is one of the most popular tools for understanding some aspects of user behavior, specifically through session recordings and heatmaps. These tools are useful because they enable you to see how users actually interact with your product or website, both at an individual and aggregate level. 
+
+However, one of the limitations of Hotjar is that it offers only a handful of such tools and doesn’t provide the deeper product analytics tools that modern teams rely on. Often this means that Hotjar is deployed alongside other tools, such as Mixpanel, Amplitude or Pendo — meaning integrations are required to share data and teams need to frequently switch between platforms. 
+
+[Hotjar is also costly on its own](https://www.hotjar.com/pricing/), often costing hundreds of pounds per month or limiting free users to only 1,050 monthly session recordings. 
+
+In this tutorial we’ll explore how PostHog can be used as an alternative to Hotjar, offering greater functionality so that teams don’t need to use multiple tools at once.
+
+## Step 1: Get started with PostHog
+
+![free hotjar alternative - start](../images/tutorials/hotjar-alternative/get-started-with-posthog.png)
+
+If you haven’t already started using PostHog, that’s the obvious first step. You can choose between self-hosted or cloud-hosted options depending on your preferences and privacy requirements. Broadly speaking self-hosted deployments require more technical knowledge to get started, but offer greater control over your data — but [check the Pricing page for more information](/pricing).
+
+Whichever option and pricing tier you choose, PostHog offers a generous free allowance of events:
+
+- Users on [PostHog Cloud](/pricing) receive 10,000 free events each month
+- Users on [PostHog Scale](/pricing) receive 1 million free events each month
+- Users on [PostHog Open Source](/pricing) have no limit on the number of tracked events
+
+It’s worth noting that some tiers do not have access to premium features, such as [Correlation Analysis](/docs/user-guides/correlation) or [A/B Experimentation](/docs/user-guides/experimentation). However, _all_ tiers have access to core features such as [Session Recording](/docs/user-guides/recordings), [Funnel](/docs/user-guides/funnels) & [Trend Analysis](/docs/user-guides/trends) and [Feature Flags](/docs/user-guides/feature-flags). 
+
+> Session Recordings do not currently count towards your free event limit and there is no limit on the number of recordings you can capture on any pricing tier.
```

Comment:
My concern with stating this - how long will we keep it free for? Session recordings is pretty expensive for us to operate and if we get a few people abusing this we'll have to put in some kind of limit - fairly quickly.

Also there's a retention limit of 29days I believe (which might be worth calling out somewhere) @rcmarron

Reply:
I get this, but I'm unaware of any definite plans to begin charging for it or limiting it at the moment - and we've recognised this loophole at least since early November, when it was first discussed. 

At the moment it does exist without limits or charges, so I personally feel we should either be closing that loophole or taking advantage of it. If we're not planning to close it...

I'll add a line about retention policy in there.

Reply:
Sod it, pushed a quick update to reflect this.